GraphicEditor/View/UserControls/CardiologyFirstFloorMapUserControl.xaml

```diff
@@ -124,14 +111,66 @@
                 <Border BorderBrush="Black"
                         BorderThickness="3"
                         Margin="806,495,255,126">
-                    <TextBox x:Name="elevator" 
-                             Background="#FF2F3B46"
-                             Text="Elevator"
-                             TextAlignment="Center"
-                             FontSize="24" 
-                             FontWeight="Bold"/>
+                    <TextBox Style="{StaticResource ElevatorTextBox}"/>
                 </Border>
 
+                <!--Decoration-->
+                <Image x:Name="image" 
+                       HorizontalAlignment="Left" 
+                       Height="44" 
+                       Margin="560,149,0,0" 
+                       VerticalAlignment="Top"
+                       Width="51"
+                       Source="/GraphicEditor;component/Resources/Icons/plant.png"/>
+
+                <!--Doors-->
+                <Rectangle Width="110"
+                           Margin="202,366,0,0" 
+                           Style="{StaticResource DoorRectangle}">
+                    <Rectangle.RenderTransform>
+                        <TransformGroup>
+                            <ScaleTransform/>
+                            <SkewTransform/>
+                            <RotateTransform Angle="90"/>
```

Comment:
Can you make this without these <ScaleTransform/> and other tags. They seem unnecesary? If not, can you maybe move these things to style like in folowing pattern: Rename "DoorRectangle" to "DoorVerticalRectangle" and make a new style based on the previous like "DoorHorizontalRectangle" but only thing that is different is this rotation?